maidsafe/safe_browser
# Web id drop down fix

resolves #809 

It resolves the issue with the webId drop throwing an error that completely blocks the Blocks the AddressBar. Also updates to the AddTabEnd reducer and test. Aswell as a minor fix to updateTabWebID and also has tests for updateTabWebID.

QA- 
The best way to test this PR is
1) Launch the browser
2) toggle Experimental Api's
3) Authenticate the browser
4) Retrieve the WebID
5) click on launch WebIdMgr to andd see if an error shows that blocks the Address Bar
